packages/map-components/templates/react/src/index.tsx

```diff
@@ -1,14 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import "./index.css";
 
 import { ArcgisMap, ArcgisSearch, ArcgisLegend } from "@arcgis/map-components-react";
-
-import "@arcgis/core/assets/esri/themes/dark/main.css";
+// import defineCustomElements to register custom elements with the custom elements registry
+import { defineCustomElements } from "@arcgis/map-components/dist/loader";
+// Register custom elements
+defineCustomElements();
 
 const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement);
 root.render(
   <React.StrictMode>
-    <ArcgisMap item-id="d5dda743788a4b0688fe48f43ae7beb9">
+    <ArcgisMap 
+      item-id="d5dda743788a4b0688fe48f43ae7beb9" 
+      onArcgisViewReadyChange={(event: any) => {
+      console.log('Map View ready', event);
```

Comment:
I might've documented this incorrectly somewhere. It should be `MapView` instead of `Map View`

Reply:
React samples should use `itemId` instead of `item-id`.